File: src/main/java/io/github/saneea/dvh/feature/time/Time.java
package io.github.saneea.dvh.feature.time;

import java.util.Map;
import java.util.function.Supplier;

import io.github.saneea.dvh.Feature;
import io.github.saneea.dvh.FeatureContext;
import io.github.saneea.dvh.feature.multi.MultiFeatureBase;

public class Time extends MultiFeatureBase {

	@Override
	public Meta meta(FeatureContext context) {
		return Meta.from("time utils");
	}

	@Override
	public Map<String, Supplier<Feature>> getFeatureAliases() {
		return new AliasesBuilder()//
				.feature("now", Now::new)//
				.feature("convert", Converter::new)//
				.feature("sleep", Sleep::new)//
				.build();
	}

}

File: src/main/kotlin/io/github/saneea/dvh/feature/time/Time.kt
package io.github.saneea.dvh.feature.time

import io.github.saneea.dvh.Feature.Meta
import io.github.saneea.dvh.FeatureContext
import io.github.saneea.dvh.feature.multi.FeatureCreator
import io.github.saneea.dvh.feature.multi.MultiFeatureBase

class Time : MultiFeatureBase() {
    override val meta = Meta("time utils")

    override fun getFeatureAliases(): Map<String, FeatureCreator> =
        AliasesBuilder()
            .feature("now", ::Now)
            .feature("convert", ::Converter)
            .feature("sleep", ::Sleep)
            .build()
}